src/s2s/mongoose_s2s_out.erl

```diff
@@ -496,11 +501,9 @@ handle_socket_packet(#s2s_data{parser = Parser} = Data, Packet) ->
 
 -spec handle_socket_elements(data(), [exml_stream:element()], non_neg_integer()) -> fsm_res().
 handle_socket_elements(#s2s_data{myname = LServer, shaper = Shaper} = Data, Elements, Size) ->
+    [execute_element_events(Element, LServer, s2s_element_in, xmpp_element_size_in)
+     || Element = #xmlel{} <- Elements],
```

Comment:
How is it possible these weren't `xmlel`?

Reply:
It's possible if it's stream opening/ending tags.

Reply:
Truuu, the opening/ending ones 👌🏽